includes/hide.php

```diff
@@ -204,13 +204,13 @@ function fastwc_should_hide_pdp_button_for_unsupported_product( $should_hide, $p
 function fastwc_should_hide_pdp_button_for_out_of_stock_product( $should_hide, $product ) {
 
 	if ( ! $should_hide ) {
-		$stock_status = method_exists( $product, 'get_stock_status' ) ? $product->get_stock_status : '';
+		$stock_status = method_exists( $product, 'get_stock_status' ) ? $product->get_stock_status() : '';
 
 		if ( 'outofstock' === $stock_status ) {
 			$should_hide = true;
 		}
 
-		fastwc_log_info( 'PDP button' . ( $should_hide ? '' : ' not' ) . ' hidden for out of stock product: ' . $product_id );
+		fastwc_log_info( 'PDP button' . ( $should_hide ? '' : ' not' ) . ' hidden for out of stock product: ' . $product->get_id() );
```

Comment:
This line should not change since we are defining `$product_id` above.

```suggestion
		fastwc_log_info( 'PDP button' . ( $should_hide ? '' : ' not' ) . ' hidden for out of stock product: ' . $product_id );
```